### DISCUSSION TITLE
Unified way to get the operating system font family

### DISCUSSION THREAD

**Discussion by cl2raul66** [CommunityToolkit/Maui#2316] [New Feature Discussions]
For a text editor, it is essential to have a complete list of the font families in the system. This allows for accurate and appropriate selection for each project, ensuring that the text is not only legible, but also has the desired style. Font families such as Serif, Sans-Serif, Cursive, Fantasy, and Monospace offer a variety of options to customize and visually enrich any document.

**Comment by cl2raul66** [comment]
I have a solution for this problem and would like to add it here, if possible.

**Reply by bijington** [reply]
Are you able to share some details on what the solution looks like please? Then we can consider how it fits into the toolkit. And also see if other members of the community are keen for it

**Reply by cl2raul66** [reply]
There is no unified method to get the OS font list, so I propose a solution that I have already developed. I'd like to include it here, since I always use CommunityToolkit.Maui in my .NET MAUI projects. If you're interested in how I implemented it, I can provide you with the library repository.

**Reply by bijington** [reply]
Please do share the repository

**Reply by cl2raul66** [reply]
Sorry for the delay, but I have a non-programming job and can only schedule in the evenings. Here's the link to the sample repository with an example application.
**Note: I've only tested this on Android and Windows devices.**
[link](https://github.com/cl2raul66/MauiFontFamily.git)
